vpr/src/power/power_components.cpp

```diff
@@ -48,7 +48,7 @@ void power_components_init() {
     int i;
     auto& power_ctx = g_vpr_ctx.mutable_power();
 
-    power_ctx.by_component.components = (t_power_usage*)vtr::calloc(POWER_COMPONENT_MAX_NUM, sizeof(t_power_usage));
+    power_ctx.by_component.components = new t_power_usage[POWER_COMPONENT_MAX_NUM]();
```

Comment:
Worth commenting that this tricky bit of code initializes all the power usage data (built in types anyway) to 0, or (probably even better) just make a constructor in t_power_usage that does that.  Then the new will call the default constructor and it will be less mysterious.